test/validation/api/ipsec/test_vectors.h

```diff
@@ -1730,6 +1735,67 @@ static const ipsec_test_packet pkt_mcgrew_gcm_test_15_esp = {
 	},
 };
 
+static const ODP_UNUSED ipsec_test_packet pkt_ipv4_rfc7634 = {
+	.len = 98,
+	.l2_offset = 0,
+	.l3_offset = 14,
+	.l4_offset = 34,
+	.data = {
+		/* ETH - not a part of RFC, added for simplicity */
+		0xf1, 0xf1, 0xf1, 0xf1, 0xf1, 0xf1,
+		0xf2, 0xf2, 0xf2, 0xf2, 0xf2, 0xf2, 0x08, 0x00,
+
+		/* IP */
+		0x45, 0x00, 0x00, 0x54, 0xa6, 0xf2, 0x00, 0x00,
+		0x40, 0x01, 0xe7, 0x78, 0xc6, 0x33, 0x64, 0x05,
+		0xc0, 0x00, 0x02, 0x05,
+
+		/* ICMP */
+		0x08, 0x00, 0x5b, 0x7a, 0x3a, 0x08, 0x00, 0x00,
+		0x55, 0x3b, 0xec, 0x10, 0x00, 0x07, 0x36, 0x27,
+		0x08, 0x09, 0x0a, 0x0b, 0x0c, 0x0d, 0x0e, 0x0f,
+		0x10, 0x11, 0x12, 0x13, 0x14, 0x15, 0x16, 0x17,
+		0x18, 0x19, 0x1a, 0x1b, 0x1c, 0x1d, 0x1e, 0x1f,
+		0x20, 0x21, 0x22, 0x23, 0x24, 0x25, 0x26, 0x27,
+		0x28, 0x29, 0x2a, 0x2b, 0x2c, 0x2d, 0x2e, 0x2f,
+		0x30, 0x31, 0x32, 0x33, 0x34, 0x35, 0x36, 0x37,
+	},
+};
+
+static const ODP_UNUSED ipsec_test_packet pkt_ipv4_rfc7634_esp = {
```

Comment:
do you need ODP_UNUSED here? structure has reference in the code.

Reply:
Yes, but not in every code that includes this header file. See other test vectors there, which also have such this attribute.